charts/trino/templates/deployment-coordinator.yaml

```diff
@@ -82,6 +82,11 @@ spec:
             - name: http
               containerPort: {{ .Values.service.port }}
               protocol: TCP
+          {{- range $key, $value := .Values.coordinator.additionalExposedPorts }}
```

Comment:
Probably for Coordinator we could reflect them on K8s Service lvl. 
But I assume you want to use those ports more for Debug so to 
actually use `kubectl port-forward` and in such case it's not so important.
But maybe let's add it just for sanity.

Reply:
I've added it to svc.